druid-shell/src/dialog.rs

```diff
@@ -33,12 +33,17 @@ pub enum FileDialogType {
 
 /// Options for file dialogs.
 #[derive(Debug, Clone, Default)]
-pub struct FileDialogOptions {
+pub struct FileDialogOptions<'a> {
```

Comment:
I expect this to cause problems? In particular I think we need to be able to `Box` this type in order to use it with `Command`.

I _believe_ this was the rationale behind the `'static` lifetimes in `FileSpec` as well.

Reply:
You're right. Lifetimes won't work here. However `'static` isn't going to work either, because these need to be able to be generated at runtime.

Could just convert them to `String` - or maybe some sort of Enum of `&'static str` and `String`? Or is there a better standard way? Surely this is a common issue.